components/board/analog_smoother.go

```diff
@@ -116,6 +117,10 @@ func (as *AnalogSmoother) Start(ctx context.Context) {
 				if errors.Is(err, errStopReading) {
 					break
 				}
+				// If the serial port was closed by the board, stop the analog smoother.
+				if strings.Contains(err.Error(), "file already closed") {
+					break
+				}
```

Comment:
Please specify this if the analog smoother is connected through and reading serial, not through a spi bus as the pi/mcp3xxx does. 

Also - let's double-check that this doesn't cause any analog reader bug with the pi/jetson and one of the mcp3008 chips once this is merged. You can make a follow-up ticket.